src/components/SearchBar/SearchBar.jsx

```diff
@@ -7,27 +9,35 @@ class SearchBar extends Component {
     super(props);
     this.state = {
       searchBy: 'title',
+      searchFor: 'Batman',
     };
   }
 
-  handelSearch(val = 'title') {
+  handleSearchBy = (val = 'title') => {
     this.setState({ searchBy: val });
-  }
+  };
+
+  handleInputChange = (e) => {
+    this.setState({ searchFor: e.target.value });
+  };
 
   render() {
+    const { searchFor, searchBy } = this.state;
     return (
       <div className={styles.searchForm}>
-        <form>
+        <div>
           <div className={styles.formTitle}>Find your movie</div>
-          <input type="email" id={styles.searchBar} placeholder="Search" />
-          <SearchBy onHandleSearch={val => this.handelSearch(val)} activeFilter={this.state.searchBy} />
-          <button className={styles.submitBtn} type="submit">
-            Search
-          </button>
-        </form>
+          <input type="text" id={styles.searchBar} placeholder="Search" value={searchFor} onChange={e => this.handleInputChange(e)} />
```

Comment:
if you have used arrow function for handleInputChange then you can directly pass it to the onChange callback: `onChange={ this.handleInputChange }`